receiver/awscontainerinsightreceiver/internal/prometheusscraper/decoratorconsumer/decorator_testutils.go

```diff
@@ -63,23 +63,25 @@ func RunDecoratorTestScenarios(ctx context.Context, t *testing.T, dc consumer.Me
 	}
 }
 
-func GenerateMetrics(nameToDimsGauge map[MetricIdentifier]map[string]string) pmetric.Metrics {
+func GenerateMetrics(nameToDimsGauges map[MetricIdentifier][]map[string]string) pmetric.Metrics {
 	md := pmetric.NewMetrics()
 	ms := md.ResourceMetrics().AppendEmpty().ScopeMetrics().AppendEmpty().Metrics()
-	for metric, dims := range nameToDimsGauge {
-		m := ms.AppendEmpty()
-		m.SetName(metric.Name)
-		metricBody := m.SetEmptyGauge().DataPoints().AppendEmpty()
-		if metric.MetricType == pmetric.MetricTypeSum {
-			metricBody = m.SetEmptySum().DataPoints().AppendEmpty()
-		}
-		metricBody.SetIntValue(10)
```

Comment:
It used to `SetIntValue(10)` and now it's `SetDoubleValue(0)`... does that affect anything?

Reply:
No it doesnt affect the other testcases , we never assert values, I added them to have a uniform value for all tests, I can make assertions for the same in out test case.